sql-mutations/insert-languages.sql

```diff
@@ -0,0 +1,2 @@
+insert into "languages" ("name")
+values ('HTML'),('CSS'),('JavaScript');
```

Comment:
You'll want `returning *` here, and in all your mutations. That's not critical now, but it will be when we do full-stack development.